packages/api/lib/request.js

```diff
@@ -35,6 +44,17 @@ async function verifyJwtAuthorization(requestJwtToken) {
   return accessToken;
 }
 
+/**
+* Validate request has header matching expected minimum version
+* @param {Request} req - express Request object
+* @param {number} minVersion - Minimum API version to allow
+* @returns {boolean}
+*/
+function isMinVersionApi(req, minVersion) {
+  const requestVersion = Number(req.headers['cumulus-api-version']);
+  return isNumber(requestVersion) && minVersion <= requestVersion;
```

Comment:
Is this `isNumber` test doing what we think it's doing? What is it going to catch?

```
> const { isNumber } = require('lodash')
undefined
> isNumber(NaN)
true
```

Reply:
`Note: To exclude Infinity, -Infinity, and NaN, which are classified as numbers, use the _.isFinite method.` 

Indeed.       Code worked/passed because NaN is unordered, but that's unintentional/side effect.   Good catch: 41e07017c8